static/app/views/monitors/components/statusToggleButton.tsx

```diff
@@ -17,7 +24,20 @@ function StatusToggleButton({monitor, ...props}: StatusToggleButtonProps) {
     ? t('Reactive this monitor')
     : t('Disable this monitor and discard incoming check-ins');
 
-  return <Button icon={<Icon />} aria-label={label} title={label} {...props} />;
+  return (
+    <Button
+      icon={<Icon />}
+      aria-label={label}
+      title={label}
+      onClick={() => onToggleStatus(isDisabeld ? 'active' : 'disabled')}
```

Comment:
nit: didn't catch this before but should be isDisabled*

Reply:
on it